packages/store/internals/src/index.ts

```diff
@@ -3,4 +3,4 @@ export { NgxsBootstrapper } from './ngxs-bootstrapper';
 export { memoize } from './memoize';
 export { ObjectUtils } from './object-utils';
 export { INITIAL_STATE_TOKEN, InitialState } from './initial-state';
-export { ObjectKeyMap, StateClass } from './symbols';
+export { ObjectKeyMap, StateClass, Callback } from './symbols';
```

Comment:
It is really not worth exposing a generic utility type like this.
Although this is an 'internal' public API it should only be used when there is no other option than to expose the internals.
Let's keep it internal and private.

Reply:
done